daemons/fenced/fenced_remote.c

```diff
@@ -1255,12 +1257,12 @@ create_remote_stonith_op(const char *client, xmlNode *request, gboolean peer)
 
     pcmk__xe_get_int(request, PCMK__XA_ST_CALLID, &(op->client_callid));
 
-    crm_trace("%s new fencing op %s ('%s' targeting %s for client %s, "
-              "base timeout %ds, %u %s expected)",
-              (peer && dev)? "Recorded" : "Generated", op->id, op->action,
-              op->target, op->client_name, op->base_timeout,
-              op->replies_expected,
-              pcmk__plural_alt(op->replies_expected, "reply", "replies"));
+    pcmk__trace("%s new fencing op %s ('%s' targeting %s for client %s, base "
+                "timeout %ds, %u %s expected)",
+                ((peer && (dev != NULL))? "Recorded" : "Generated"), op->id,
```

Comment:
In other spots, you've broken out more complex conditionals like this into a separate boolean variable and then checked it in the commit message.  This would be a good place to do that too.

Reply:
Adding a commit to `CRM_CHECK(dev != NULL, return NULL)` at the beginning.